crates/dbs-arch/src/aarch64/regs.rs

```diff
@@ -111,20 +111,20 @@ arm64_sys_reg!(MPIDR_EL1, 3, 0, 0, 0, 5);
 /// * `mem` - Reserved DRAM for current VM.
 pub fn setup_regs(vcpu: &VcpuFd, cpu_id: u8, boot_ip: u64, fdt_address: u64) -> Result<()> {
     // Get the register index of the PSTATE (Processor State) register.
-    vcpu.set_one_reg(arm64_core_reg!(pstate), PSTATE_FAULT_BITS_64)
+    vcpu.set_one_reg(arm64_core_reg!(pstate), PSTATE_FAULT_BITS_64 as u128)
```

Comment:
Is it possible to modify the definition directly to 128 bits?

Reply:
"set/get_one_reg" interface is defined in [rust-vmm/kvm-ioctls](https://github.com/rust-vmm/kvm-ioctls/), which enables 128-bit registers because some registers on aarch64 is 128-bit. This change is released in kvm-ioctls 0.12.0 version.
It is safe to convert 64-bit value to 128-bit format directly here.